internal/util/paramtable/base_table.go

```diff
@@ -412,12 +416,23 @@ func (gp *BaseTable) SetLogger(id UniqueID) {
 	}
 }
 
+func (gp *BaseTable) loadKafkaConfig() {
+	brokerList := os.Getenv("KAFKA_BROKER_LIST")
+	if brokerList == "" {
+		brokerList = gp.Get("kafka.brokerList")
+	}
+	gp.Save("_KafkaBrokerList", brokerList)
+}
+
 func (gp *BaseTable) loadPulsarConfig() {
 	pulsarAddress := os.Getenv("PULSAR_ADDRESS")
 	if pulsarAddress == "" {
-		pulsarHost := gp.LoadWithDefault("pulsar.address", DefaultPulsarHost)
-		port := gp.LoadWithDefault("pulsar.port", DefaultPulsarPort)
-		pulsarAddress = "pulsar://" + pulsarHost + ":" + port
+		pulsarHost := gp.Get("pulsar.address")
```

Comment:
what's the reason why we not support gp.LoadWithDefault?

Reply:
gp.LoadWithDefault must set a default value,   it needs to be optional once Milvus has supported Kafka